sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerAsyncClient.java

```diff
@@ -971,8 +972,8 @@ public String generateSAS(String identifier, ContainerSASPermission permissions,
     /**
      * Sets blobServiceSASSignatureValues parameters dependent on the current blob type
      */
-    private BlobServiceSASSignatureValues configureServiceSASSignatureValues(
-        BlobServiceSASSignatureValues blobServiceSASSignatureValues, String accountName) {
+    private BlobServiceSasSignatureValues configureServiceSASSignatureValues(
```

Comment:
nit: configureServiceSASSignatureValues -> configureServiceSasSignatureValues and same with parameter name.

Reply:
Yes, I will have a separate PR for renaming all acronym related classes to minimize merge conflicts.